scripts/nginx/requestrr.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Nginx Configuration for requestrr
+master=$(cut -d: -f1 < /root/.master.info)
```

Comment:
We actually have a function for this now which should be available without having to source it, should be mentioned in `sources/globals.sh`